discovery/cmd/stub_test.go

```diff
@@ -45,6 +45,31 @@ func TestClientStub(t *testing.T) {
 	require.Contains(t, err.Error(), "Unimplemented desc = unknown service discovery.Discovery")
 }
 
+func TestClientStubEd25519(t *testing.T) {
+	srv, err := corecomm.NewGRPCServer("127.0.0.1:", corecomm.ServerConfig{
+		SecOpts: corecomm.SecureOptions{},
+	})
+	require.NoError(t, err)
+	go srv.Start()
+	defer srv.Stop()
+
+	_, portStr, _ := net.SplitHostPort(srv.Address())
+	endpoint := fmt.Sprintf("localhost:%s", portStr)
+	stub := &ClientStub{}
+
+	req := discovery.NewRequest()
+
+	_, err = stub.Send(endpoint, common.Config{
+		SignerConfig: signer.Config{
+			MSPID:        "Org1MSP",
+			KeyPath:      filepath.Join("testdata", "ed25519_sk"),
+			IdentityPath: filepath.Join("testdata", "ed25519.pem"),
+		},
+		TLSConfig: comm.Config{},
+	}, req)
+	require.Contains(t, err.Error(), "Unimplemented desc = unknown service discovery.Discovery")
```

Comment:
but why would we want to test this case? I think it's not a very interesting case to demonstrate compatibility with ed25519. 

The discovery code base has nothing to do with the signature algorithm. I don't think we should be expanding these unit tests for this PR.

Reply:
I was lost as to what specifically test, so I probably implemented it due to the presence of a _SignerConfig_ loading ECDSA key/certificate. I will remove it.